src/main/java/com/google/sps/servlets/PersonServlet.java

```diff
@@ -38,7 +39,7 @@ public void init(PersonDao personDao) {
     this.personDao = personDao;
   }
 
-  // Sends the request's contents to Datastore in the form of a new Person
+  // Sends the request's contents to Datastore in the form of a new Person.
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
What's the context that `post` is used?

I'm trying to discern whether you're planning to allow arbitrary people to `post` a new user. If so, that's suspicious.

Reply:
lt's what happens after someone registers. Should l check if the request has all the necessary fields?

Reply:
Hmm..

I'm not sure exactly what precautions make sense. Perhaps for now put a todo to "consider further restricting access to this API"?